src/Agent.Worker/Release/Artifacts/Definition/IArtifactDetails.cs

```diff
@@ -0,0 +1,16 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="IArtifactDetails.cs" company="Microsoft Corporation">
+//   2012-2023, All rights reserved.
+// </copyright>
+// <summary>
+//   Defines the IArtifactDetails type.
+// </summary>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace Agent.Worker.Release.Artifacts.Definition
+{
+    public interface IArtifactDetails
+    {
+        string RelativePath { get; set; }
```

Comment:
The relativepath is still send by the server but probably for backcompat.. will find out and fix them if required..